recipe/meta.yaml

```
@@ -32,11 +32,26 @@ requirements:
     - tqdm >=4.42.1
     - typing-extensions >=3.7.4.3
   run_constrained:
+    # cli
+    - InquirerPy ==0.3.4
```

```suggestion
    - inquirerpy ==0.3.4
```
see https://pypi.org/project/inquirerpy/